docs/Lua API.rst

```diff
@@ -2984,6 +2984,80 @@ Predefined instance methods:
 
 To avoid confusion, these methods cannot be redefined.
 
+.. _custom-raw-tokens:
+
+custom-raw-tokens
+===============
```

Comment:
This is causing the build error due docs: home/runner/work/dfhack/dfhack/docs/Lua API.rst:2990: WARNING: Title underline too short.